### PR TITLE
Translate documents related in the system token into english

### PR DESCRIPTION
**Pull request by energyGiver**
# Overview
- system-token
- transction-fee-model
- open-message
- transfer-assets
- how-to-interact-with-system-token
- how-to-pay-transaction-fee

### CODE REVIEW

File: en/infrablockchain/learn/protocol/system-token.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
-title: 시스템 토큰
-description: 시스템 토큰에 대한 전반적인 내용을 다룹니다.
+title: System Tokens
```
Comment (cocoyoon) [inline en/infrablockchain/learn/protocol/system-token.md]: 's'

**Review by cocoyoon — APPROVED** [review]

File: en/infrablockchain/tutorials/build/transfer-assets-with-xcm.md [hunk 2]
```diff
@@ -2,153 +2,156 @@
 title: XCM을 이용하여 원격으로 자산 전송하기
```
Comment (sweatpotato13) [inline en/infrablockchain/tutorials/build/transfer-assets-with-xcm.md]: 타이틀 번역 안됨

**Review by sweatpotato13 — CHANGES_REQUESTED** [review]
🐧

File: en/infrablockchain/tutorials/build/transfer-assets-with-xcm.md (hunk 2 above)
Comment (energyGiver) [reply en/infrablockchain/tutorials/build/transfer-assets-with-xcm.md]: great!

**Review by sweatpotato13 — APPROVED** [review]
👍

**Pull request merged**